### PR TITLE
Homework stl ng90

### PR DESCRIPTION
**Pull request by NG90**

### CODE REVIEW

File: module1/homework/grayscaleImages/compression.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+void printMap(const std::array<std::array<uint8_t, width>, height>& bitmap) {
+    for (const auto& row : bitmap) {
+        for (const auto el : row) {
+            if (std::isprint(el)) {
+                std::cout << el;
+            } else {
+                std::cout << ' ';
+            }
```
Comment (NG90) [inline module1/homework/grayscaleImages/compression.cpp:52]: TODO: Change to:
`std::cout << std::isprint(el) ? el : ' '`

File: module1/homework/grayscaleImages/compression.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#include <cctype>
+#include <iostream>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(
+    const std::array<std::array<uint8_t, width>, height>& bitmap) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressedBitmap{};
+
+    for (const auto& el : bitmap) {
+        compressedBitmap.push_back(std::make_pair(*begin(el), 1));
+        for (auto iter = std::next(begin(el)); iter != end(el); ++iter) {
```
Comment (NG90) [inline module1/homework/grayscaleImages/compression.cpp:12]: TODO: const iterator

File: module1/homework/lengthSort/sort.cpp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& listToSort) {
+    std::deque<std::string> output{begin(listToSort), end(listToSort)};
```
Comment (cnuebred) [inline module1/homework/lengthSort/sort.cpp:6]: hey :)
Can you tell me what does it mean? Why after output in {} you call begin and end of listToSort?

Comment (NG90) [reply module1/homework/lengthSort/sort.cpp:6]: This way deque is constructed based on the elements from list, so elements from list are simply copied to deque.

**Pull request merged**